src/server/models/matching.model.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let matchingSchema = new Schema({
+    userId: String,
```

Comment:
I don't think I understand this struct.
At first I thought this was the mentor/mentee matching relationship, but that stuff seems to be in relationship.model.js